scipy/meson.build

```diff
@@ -67,6 +68,15 @@ lapack = dependency('openblas')
 #       _build_utils/_fortran.py)
 g77_abi_wrappers = files('_build_utils' / 'src' / 'wrap_dummy_g77_abi.f')
 
+python_exe = find_program('python3')
```

Comment:
This should not be needed. Instead, use the Python detected in the top-level `meson.build` (I think you can just use it as `py3` here.

Reply:
Sure. It should be fixed now.